lte/cloud/go/services/lte/client_api.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package lte
+
+import (
+	"context"
+	"fmt"
+
+	"magma/lte/cloud/go/lte"
+	"magma/lte/cloud/go/serdes"
+	lte_models "magma/lte/cloud/go/services/lte/obsidian/models"
+	"magma/lte/cloud/go/services/lte/protos"
+	"magma/orc8r/cloud/go/serde"
+	merrors "magma/orc8r/lib/go/errors"
+	"magma/orc8r/lib/go/registry"
+
+	"github.com/golang/glog"
+)
+
+func GetEnodebState(networkID string, gatewayID string, enodebSN string) (*lte_models.EnodebState, error) {
+	client, err := getClient()
+	if err != nil {
+		return nil, err
+	}
+	res, err := client.GetEnodebState(
+		context.Background(),
+		&protos.GetEnodebStateRequest{
+			NetworkId: networkID,
+			GatewayId: gatewayID,
+			EnodebSn:  enodebSN,
+		},
+	)
+	if err != nil {
+		return nil, err
+	}
+	enodebStateI, err := serde.Deserialize(res.SerializedState, lte.EnodebStateType, serdes.State)
+	if err != nil {
+		return nil, fmt.Errorf("could not deserialize enodeb state for enodeb %s", enodebSN)
```

Comment:
Small nit here and elsewhere: I prefer `error deserializing` rather than `could not deserialize` or `failure to deserialize`. Not the end of the world either way, but aligning on the former (a) provides uniformity in the codebase and (b) is always greppable with the "error" keyword, regardless of context

That said, removing the "could not" or "error" etc is ideal where reasonable. Since this is ~top-level (client api), it probably makes sense to include the "error" keyword here. But just general thoughs https://github.com/uber-go/guide/blob/master/style.md#error-wrapping